legacy/src/api/assets/tokens.md

```diff
@@ -70,3 +70,110 @@ Token value may be set in multiple currencies and is the same for all tokens in
 | :----- | :-------------------- | :----------------------------------------------------------------------- |
 | 403    | TOKEN_ALREADY_CREATED | Token with supplied parameters already exists.                           |
 | 403    | LIVENESS_MISMATCH     | The account is test and the collection's liveness is main or vice versa. |
+
+## Create Token Collection
```

Comment:
When viewing these docs in the browser it might be more intuitive to have the Create Token Collection first, then Create Redemption Condition, and then Create Token?

Reply:
We should also make the link between tokens and collections clearer. E.g. for the create token API, we could link to the collections section when talking about collectionId.